.vscode/launch.json

```diff
@@ -25,9 +24,8 @@
             "name": "Launch Program",
             "program": "${workspaceFolder}/bin/checksync.dev.js",
             "args": [
-                "-m",
-                ".ka_root",
-                "/Users/jeffyates/khan/webapp/Makefile"
+                "--config",
+                "/Users/jeffyates/khan/webapp/.checksyncrc.json"
```

Comment:
Hmm this doesn't seem right, it's pinned to your home directory?

Reply:
Yeah, it was before too when it pointed at the Makefile. I couldn't get it working nice any other way. I'm OK with it for now; it's just me deving this stuff.